docs/patrol/alt-native-setup.mdx

```diff
@@ -0,0 +1,298 @@
+---
+title: Patrol - setup
+---
+
+# Native automation setup
+
+Using Patrol's native automation feature requires more setup work than using
+custom finders. Unfortunately it's necessary, because you have to integrate your
+app with Android and/or iOS native testing tools.
+
+We believe that the powerful features you'll get will make up for it!
+
+<Warning>
+        In this tutorial, we are using example app, which has package name
+        `com.example.myapp` on Android, bundle id `com.example.MyApp` on iOS
+        and `My App` name on both Android and iOS. Replace any occurences
+        of those names with proper values.
+</Warning>
+
+### Add dependency on patrol
+
+If you haven't already, add a dependency on the `patrol` package in the
+`dev_dependencies` section of **pubspec.yaml**.
+
+```
+flutter pub add patrol --dev
+```
+
+### Configure Patrol in pubspec.yaml
+
+Create `patrol` section in your **pubspec.yaml**:
+
+```yaml title="pubspec.yaml"
+dependencies:
+  # ...
+
+dev_dependencies:
+  # ...
+
+patrol:
+  app_name: My App
+  android:
+    package_name: com.example.myapp
+  ios:
+    bundle_id: com.example.MyApp
+```
+
+### Install patrol_cli
+
+Patrol CLI (command-line interface) is a small program that enables running
+Patrol UI tests. It is necessary to run UI tests (`flutter test` won't work!).
```

Comment:
I would add a warning tile informing that `flutter test` won't work

Reply:
I think people tend to skip this tiles, but on the other hand if such a tile is short, then it might be a good idea